hooks/useRailgunTx.tsx

```diff
@@ -39,6 +45,41 @@ const useRailgunTx = () => {
     return shieldToken(args);
   };
 
+  const getGasDetailsSerialized = async (
+    gasEstimateString: string
+  ): Promise<TransactionGasDetailsSerialized> => {
+    const { maxFeePerGas, maxPriorityFeePerGas, gasPrice } = await provider
+      .getFeeData()
+      .then((data) => {
+        // polygon maxPriorityFee must be at least 30
+        if (network === NetworkName.Polygon) {
+          const gwei30 = parseUnits("30", "gwei");
+          return {
+            ...data,
+            maxPriorityFeePerGas: data.maxPriorityFeePerGas!.lt(gwei30)
+              ? gwei30
+              : data.maxPriorityFeePerGas,
+          };
+        }
```

Comment:
Let's delete this (and any other related) special case polygon handling: we don't currently support polygon as discussed, and when we do support, this will be wrong anyway

Reply:
Yea I'm fine with

```typescript
if (network === NetworkName.Polygon) {
 // TODO
}
```

or something like that. But right now the code makes it seem like the current implementation is correct, which is why I think we should remove it

Reply:
will do, thanks!

Reply:
2d9ced1